generate-public-site.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+mkdir -p site/public
+echo "Public topic visualization demo" >> site/public/index.html  
```

Comment:
Is it okay that you add a line to the end of `index.html` every time the script runs?

Reply:
I think it's fine, as the file does not exist on the build agent as the script is triggered. I'll however fix it in a minute.